EligibilityCheck/locks.ts

```diff
@@ -0,0 +1,54 @@
+import { Either, left, right } from "fp-ts/lib/Either";
+import { fromNullable } from "fp-ts/lib/Option";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  IResponseErrorInternal,
+  ResponseErrorForbiddenNotAuthorized,
+  ResponseErrorInternal
+} from "italia-ts-commons/lib/responses";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { BonusCode } from "../generated/models/BonusCode";
+
+import * as t from "io-ts";
+
+export const ProcessingBonus = t.type({
```

Comment:
`ProcessingBonusForUser` ?

Reply:
there can be only one processing bonus for each user, right?

Reply:
yes that's the reason of this PR (I've removed this type which is repeated anyway)